repositories.yaml

```diff
@@ -1381,3 +1381,23 @@ applications:
     moz_pipeline_metadata_defaults:
       expiration_policy:
         delete_after_days: 760
+
+  - app_name: moso_mastodon_backend
+    canonical_app_name: Mozilla.Social Mastodon Backend
+    app_description: |
+      Mastodon instance for Mozilla.Social
+    url: https://github.com/MozillaSocial/mastodon
+    notification_emails:
+      - breyes@mozilla.com
+    branch: main
+    metrics_files:
+      - .glean/metrics.yaml
+    ping_files: []
```

Comment:
Should there be a ping file in the repo?

Reply:
We are leveraging the 'events' ping which does not require a `pings.yaml` file per my understanding.  @Dexterp37 and @badboy can probably validate that for us.

Reply:
@scholtzan @braunreyes yes, that is correct (we do the same for the Glean Debug Ping Viewer for the same reason).